documentation/howtos.md

```diff
@@ -4,58 +4,125 @@ By default, instances are not provisioned with either ecommerce or the Course Di
 1. In the ecommerce env, add a Site, Partner, and Site Configuration as per the
    instructions in the [edX ecommerce docs](http://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/ecommerce/install_ecommerce.html#add-another-site-partner-and-site-configuration).
    Use the partner code from `ECOMMERCE_PAYMENT_PROCESSOR_CONFIG`.
+   Eg.
+
+   ```
+   sudo -u ecommerce -Hs
+   cd
+   source ecommerce_env
+   source venvs/ecommerce/bin/activate
+   cd ecommerce
+   python manage.py create_or_update_site \
+     --site-name 'My Site E-Commerce' \
+     --site-domain 'ecommerce.lms.external.domain' \
+     --partner-code 'partn_id' \ # this is limited to only a few characters
+     --partner-name 'Partner Name' \
+     --lms-url-root 'https://lms.external.domain' \
+     --client-id '{ecommerce_worker oauth client id}' \
+     --client-secret '{ecommerce_worker oauth client secret}' \
+     --from-email 'noreply@todo.lms.external.domain' \ # TODO: what should this be?
+     --discovery_api_url 'https://discovery.external.lms.domain'
+   ```
 1. In the discovery env, configure a partner using
    [`create_or_update_partner`](https://github.com/edx/course-discovery/blob/master/course_discovery/apps/core/management/commands/create_or_update_partner.py).
    Use the same partner code as what you used for ecommerce.
+
+   ```
+   sudo -u discovery -Hs
+   cd
+   source discovery_env
+   source venvs/discovery/bin/activate
+   cd discovery
+   # TODO: what should --site-domain be here?
+   python manage.py create_or_update_partner \
+     --site-id 1 \
+     --site-domain 'external.lms.domain' \
+     --code 'partn_id' \
+     --name 'Montessori' \
+     --courses-api-url 'https://external.lms.domain/api/courses/v1/' \
+     --ecommerce-api-url 'https://ecommerce.external.lms.domain/api/v2/' \
+     --organizations-api-url 'https://external.lms.domain/api/organizations/v0/' \
+     --oidc-url-root 'https://external.lms.domain/oauth2' \
+     --oidc-key '{discovery oauth client id}' \
+     --oidc-secret '{discovery oauth client secret}'
+   ```
 1. [Configure LMS to use ecommerce](http://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/ecommerce/install_ecommerce.html#switch-from-shoppingcart-to-e-commerce)
 
 Test your configuration:
 
 1. Verify the OIDC login for both services:
 
-        https://ecommerce-<your-instance>.opencraft.hosting/login
-        https://discovery-<your-instance>.opencraft.hosting/login
+        https://ecommerce.<your-instance>.opencraft.hosting/login
+        https://discovery.<your-instance>.opencraft.hosting/login
 
 1. Verify that the discovery cronjob runs without errors:
 
-        sudo -s -c . /edx/app/discovery/discovery_env; /edx/bin/manage.discovery refresh_course_metadata
+        sudo -u discovery -Hs
+        source /edx/app/discovery/discovery_env
+        /edx/bin/manage.discovery refresh_course_metadata
 
```

Comment:
Worth referencing the e-commerce usage docs here too: https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/ecommerce/create_products/index.html